src/Command/Cache/DoctrineResultCacheWarmerCommand.php

```diff
@@ -308,31 +256,35 @@ private function initializeBatchSizeOption(InputInterface $input): void
      */
     private function initializeLineItemIdsOption(InputInterface $input): void
     {
-        $this->lineItemIds = array_filter(
-            explode(',', (string)$input->getOption(self::OPTION_LINE_ITEM_IDS)),
-            'is_numeric'
+        $this->lineItemSlugs = array_filter(
+            explode(',', (string)$input->getOption(self::OPTION_LINE_ITEM_SLUGS)),
+            static function ($value) {
+                return !empty($value) && is_string($value);
+            }
         );
 
-        if (empty($this->lineItemIds)) {
+        if (empty($this->lineItemSlugs)) {
             throw new InvalidArgumentException(
-                sprintf("Invalid '%s' option received.", self::OPTION_LINE_ITEM_IDS)
+                sprintf("Invalid '%s' option received.", self::OPTION_LINE_ITEM_SLUGS)
             );
         }
     }
 
     /**
      * @throws InvalidArgumentException
      */
-    private function initializeUserIdsOption(InputInterface $input): void
+    private function initializeUsernamesOption(InputInterface $input): void
     {
-        $this->userIds = array_filter(
-            explode(',', (string)$input->getOption(self::OPTION_USER_IDS)),
-            'is_numeric'
+        $this->usernames = array_filter(
+            explode(',', (string)$input->getOption(self::OPTION_USERNAMES)),
+            static function ($value) {
```

Comment:
maybe specify the return type on the anonymous functions

Reply:
Totally, fixed.